sdcm/cluster_k8s/__init__.py

```diff
@@ -184,6 +189,38 @@ def deploy_loaders_cluster(self, config: str) -> None:
         self.kubectl("get statefulset", namespace="sct-loaders")
         self.kubectl("get pods", namespace="sct-loaders")
 
+    @log_run_info
+    def deploy_monitoring_cluster(self, tag: str, namespace: str = "monitoring") -> None:
```

Comment:
add a comment that we implemented the recommended procedure from the website and `dashboards.sh` can potentially fail